application/src/main/java/org/togetherjava/tjbot/commands/basic/VcActivityCommand.java

```diff
@@ -45,6 +45,12 @@
     private static final String MAX_USES_OPTION = "max-uses";
     private static final String MAX_AGE_OPTION = "max-age";
 
+    /**
+     * Sets the MAX_AGE duration of the voice channel in seconds - converts to days.
+     */
```

Comment:
The comment is kinda obsolete and doesnt add anything to the code, I would remove it.